api/nodes/views.py

```diff
@@ -179,13 +241,105 @@ def perform_create(self, serializer):
 
 
 class NodeDetail(generics.RetrieveUpdateDestroyAPIView, NodeMixin):
-    """Projects and component details.
+    """Details about a given node (project or component). *Writeable*.
 
     On the front end, nodes are considered 'projects' or 'components'. The difference between a project and a component
-    is that a project is the top-level node, and components are children of the project. There is also a category field
-    that includes the option of project. The categorization essentially determines which icon is displayed by the
-    Node in the front-end UI and helps with search organization. Top-level Nodes may have a category other than
-    project, and children nodes may have a category of project.
+    is that a project is the top-level node, and components are children of the project. There is also a [category
```

Comment:
It seems reasonable to put general discussion of nodes here and here only. We can point users to the NodeDetail docs elsewhere that it's appropriate.